string object_name
string location
---
bool success
